.github/workflows/release.yml

```diff
@@ -138,4 +142,94 @@ jobs:
           git commit -am "Hazelcast Command Line ${{ env.RELEASE }} release"
 
       - name: Push to homebrew-hz repo
-        run: git push
\ No newline at end of file
+        run: git push
+
+  verify-deb:
+    runs-on: ubuntu-latest
+    needs: build
+    name: CLI Release DEB Verification
+    steps:
+      - uses: actions/checkout@v2.3.2
+
+      - name: Install CLI from deb
+        run: |
+          # await some time for deb metadata to be signed
+          sleep 60
+          wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=hazelcast | sudo apt-key add -
+          echo "deb http://dl.bintray.com/hazelcast/deb stable main" | sudo tee -a /etc/apt/sources.list
+          sudo apt update && sudo apt install hazelcast=${{ needs.build.outputs.release }}
+          hz start -c src/test/resources/integration-test-hazelcast.yaml &
+
+      - name: Check IMDG health
+        run: |
+          attempts=0
+          max_attempts=10
+          until $(curl --silent --fail "127.0.0.1:5701/hazelcast/health/ready"); do
+            if [ ${attempts} -eq ${max_attempts} ];then
+                echo "Hazelcast not responding"
+                exit 1
+            fi
+            printf '.'
+            attempts=$(($attempts+1))
+            sleep 1
+          done
+          kill -9 $(jps | grep "HazelcastMember" | awk '{print $1;}')
```

Comment:
Nit. Alternatively you could `killall -9 java`, but up to you.

Reply:
Yes, actually removed all these kill commands since the runners are killed just after the verification runs.